### DISCUSSION TITLE
Rule to suggest using SvelteKit response helpers

### DISCUSSION THREAD

**Discussion by budivoogt** [sveltejs/eslint-plugin-svelte#939] [Ideas]
Hi there,

Novice ESLint user here. Love the library.

My collaborator and I regularly notice ourselves using default Javascript Response objects, instead of the helper functions like SvelteKit's `[json()](https://svelte.dev/docs/kit/@sveltejs-kit#json)`, `text()` and such. Would it be possible to create a rule that warns us of these, so we are encouraged to follow the SvelteKit convention? 

Thanks.

**Comment by baseballyama** [comment]
How about using the `no-restricted-syntax` rule?

https://eslint.org/play/#eyJ0ZXh0IjoiLyogZXNsaW50IG5vLXJlc3RyaWN0ZWQtc3ludGF4OiBbXCJlcnJvclwiLCBcIk5ld0V4cHJlc3Npb25bY2FsbGVlLm5hbWU9J1Jlc3BvbnNlJ11cIl0gKi9cblxuYXN5bmMgZnVuY3Rpb24gaGFuZGxlUmVxdWVzdChyZXF1ZXN0KSB7XG4gICAgY29uc3QgcmVzdWx0ID0gYXdhaXQgZmV0Y2gocmVxdWVzdCk7XG4gICAgY29uc3QgZGF0YSA9IGF3YWl0IHJlc3VsdC5qc29uKCk7IC8vIHVzaW5nIGRlZmF1bHQgSmF2YVNjcmlwdCBtZXRob2QgZGlyZWN0bHlcbiAgICByZXR1cm4gbmV3IFJlc3BvbnNlKEpTT04uc3RyaW5naWZ5KGRhdGEpLCB7XG4gICAgICAgIGhlYWRlcnM6IHsgJ0NvbnRlbnQtVHlwZSc6ICdhcHBsaWNhdGlvbi9qc29uJyB9XG4gICAgfSk7XG59Iiwib3B0aW9ucyI6eyJydWxlcyI6e30sImxhbmd1YWdlT3B0aW9ucyI6eyJzb3VyY2VUeXBlIjoic2NyaXB0IiwicGFyc2VyT3B0aW9ucyI6eyJlY21hRmVhdHVyZXMiOnt9fX19fQ==
